opensubdiv/CMakeLists.txt

```diff
@@ -366,6 +366,43 @@ if (NOT NO_LIB)
         endif()
 
     endif()
+
+    # Expose preprocessor macro to the interface so that client know which feature are enabled
```

Comment:
This portion is useful so that client consuming this any OpenSubdiv library can introspect which feature are enabled directly on the exported CMake targets.

For example, USD would benefit from this as they are checking if header exists and recompute those exact preprocessor macros because they are not exported
cf. https://github.com/PixarAnimationStudios/USD/blob/3b097e3ba8fabf1777a1256e241ea15df83f3065/cmake/modules/FindOpenSubdiv.cmake#L110-L117